power/nut-client/nut-client.yaml

```diff
@@ -0,0 +1,54 @@
+name: nut-client
+depends:
+  - service: cri
+  - network:
+    - addresses
+    - connectivity
+    - etcfiles
+container:
+  entrypoint: ./upsmon
+  args:
+    - -F
+  mounts:
+    - source: /lib
+      destination: /lib
+      type: bind
+      options:
+        - bind
+        - ro
+    - source: /usr/lib
+      destination: /usr/lib
+      type: bind
+      options:
+        - bind
+        - ro
+    # nut shared libraries
+    - source: /usr/local/lib
```

Comment:
it seems you're already installing to the container rootfs https://github.com/siderolabs/extensions/pull/67/files#diff-ed82d9600cff54c56b3a47bd58b81fb45a2c5c21322b711aa0d7f9e8bf1e81c1R76

so this mount is reduntant

Reply:
the root for the extension service is `/usr/local/lib/containers/nut-client`, I moved the libs there instead of using the bind mount